README.md

```diff
@@ -1 +1,195 @@
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>HTML</summary>
+  <ul>
+    <li>HTML</li>
+  </ul>
+</details>
+
+<details>
+  <summary>CSS</summary>
+  <ul>
+    <li>CSS</li>
+  </ul>
+</details>
+
+<details>
+<summary>Flexbox</summary>
+  <ul>
+    <li>Flexbox</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+
+- **Creating branches on repositories.**
+- **Use of linter on HTML&CSS project.**
+- **Write a correct descriptive README file.**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Create a repo on your repositores files.
+- Clone or make a copy of this repo on your local machine.
+- Follow GitHub flow.
+- A carefully reading of this README.md is required.
+
+
+### Setup
+
+Clone this repository to your desired folder:
+ 
+ ```bash
+ cd my-folder
+ git clone git@github.com:oovillagran/Portfolio-project.git
+```
+
+### Install
+
+Install this project with:
+
+ ```bash
+ npm install
+```
+
+### Usage
+
+To run the project, you can use your favorite browser.
+
+
+### Run tests
+
+To run tests, execute the following command:
+
+```bash
+ npx hint .
+```
+
+### Deployment
+
+- N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+
+👤 **Author1**
+
+- GitHub: [@oovillagran](https://github.com/oovillagran)
+- Twitter: [@oovillagran](https://twitter.com/oovillagran)
+- LinkedIn: [linkedin.com/in/oscar-villagran-91b13859](https://linkedin.com/in/oscar-villagran-91b13859)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+
+
+- [ ] **Use of linter in HTML&CSS projects**
+- [ ] **Flexbox tools are requiered**
+- [ ] **Use README template file**
+- [ ] **Ask for a code review**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+
+If you like this project feel free to make any comment, all contributions are welcome!.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+📝 License
```

Comment:
- [ ] This line should be a heading and in markdown language, we specify the heading by adding the symbol **"#"** space then your title, Please fix the License style here.